src/pages/ReimbursementAccount/BankAccountManualStep.js

```diff
@@ -97,6 +97,8 @@ class BankAccountManualStep extends React.Component {
                         defaultValue={ReimbursementAccountUtils.getDefaultStateForField(this.props, 'routingNumber', '')}
                         keyboardType={CONST.KEYBOARD_TYPE.NUMBER_PAD}
                         disabled={shouldDisableInputs}
+                        autoFocus
+                        shouldDelayFocus
```

Comment:
Thanks for the ping. I missed the notification earlier.

Reply:
Should I do it like I suggested [here](https://github.com/Expensify/App/issues/11423#issuecomment-1276518249)

Reply:
I think you can pass prop as true in platform dependent file and keep that prop as false by default in common file

https://github.com/Expensify/App#platform-specific-file-extensions

Reply:
Sorry for the inconvenience. I have raised a new [PR](https://github.com/Expensify/App/pull/12513) because I can't make changes locally to this branch, I am getting data stream error.

Reply:
No problem. Then let's close this PR